### PR TITLE
Add skip navigation link

### PR DESCRIPTION
**Pull request by derick-montague**
- Add watchers to change focus on ApplicationHeader component element.
Thisis needed because SPAs do not perform a full page load.

- Add styling for skip to content button

### CODE REVIEW

**Review by dixsie — COMMENTED** [review]
✅ Tested in Chrome with screen reader

Skip link isn't working in Firefox or Safari

**Comment by derick-montague** [comment]
Expected testing behavior. When you navigate between pages, you should be able to tab and have the `Skip to content` link transition on to the page. By hitting enter the link should transition off the screen and the next tab should be to the first focusable item on the page.

**Comment by derick-montague** [comment]
> ✅ Tested in Chrome with screen reader
> 
> Skip link isn't working in Firefox or Safari

Update your System Preferences
https://www.weba11y.com/blog/2014/07/07/keyboard-navigation-in-mac-browsers/

File: src/layouts/AppLayout.vue [hunk 1]
```diff
@@ -24,6 +24,28 @@ export default {
   components: {
     AppHeader,
     AppNavigation
+  },
+  watch: {
+    $route: function() {
+      // $nextTick = DOM updated
+      this.$nextTick(function() {
+        // set timeout needs a slight delay before referencing the DOM
+        setTimeout(() => {
```
Comment (yoshiemuranaka) [inline src/layouts/AppLayout.vue]: settimeout 0 looks a little strange to me. Did you run into issues without it? Doesn't the $nextTick callback take care of DOM not being fully rendered?

**Comment by derick-montague** [comment]
You good now, @yoshiemuranaka?

**Review by dixsie — APPROVED** [review]

**Review by yoshiemuranaka — APPROVED** [review]

**Comment by derick-montague** [comment]
Integrated to webui-vue in https://gerrit.openbmc-project.xyz/c/openbmc/webui-vue/+/28610

**Pull request merged**